pandas/core/groupby/ops.py

```diff
@@ -490,21 +495,26 @@ def _call_cython_op(
         orig_values = values
 
         dtype = values.dtype
-        is_numeric = is_numeric_dtype(dtype)
+        dtype_kind = dtype.kind
```

Comment:
for my own edification, how big a difference does this make?  i.e. should i get into the habit of doing this?